insights/client/constants.py

```diff
@@ -78,7 +78,8 @@ class InsightsConstants(object):
     sig_kill_bad = 101
     cached_branch_info = os.path.join(default_conf_dir, '.branch_info')
     pidfile = os.path.join(os.sep, 'var', 'run', 'insights-client.pid')
-    egg_release_file = os.path.join(os.sep, 'var', 'tmp', 'insights-client', 'insights-client-egg-release')
+    insights_tmp_path = os.path.join(os.sep, 'var', 'tmp', 'insights-client')
+    egg_release_file = os.path.join(insights_tmp_path, 'insights-client-egg-release')
```

Comment:
If I wanted to have exceedingly atomic changesets, I’d do this in a separate follow-up pull request. I am just making a note here, not an actual suggestion.

```suggestion
    egg_release_file = os.path.join(os.sep, 'var', 'tmp', 'insights-client', 'insights-client-egg-release')
```